app/views/dashboard/_node_wiki.html.erb

```diff
@@ -1,6 +1,12 @@
 <div class="col-lg-6 note-container-wiki">
   <div class="note note-pane note-wiki" data-index="<%= index %>">
-
+    
+    <% if current_user && node.is_new_to_user(node, current_user) %>
```

Comment:
The usage here will raise the error as node in this scenario is an object of revision. 

Add delegation in revision to use the method from node https://simonecarletti.com/blog/2009/12/inside-ruby-on-rails-delegate/


Also, you don't have to paas node as parameter when you're calling using the related object of it.